includes/polling/os/ibmc.inc.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$oids = array (
+    'version' => 'firmwareVersion.11.65.99.116.105.118.101.32.105.66.77.67',   // firmwareVersion."Active iBMC"
```

Comment:
I did not manage to use the string indexed form of the OID, so I had to write it numerically. If you have a working syntax for firmwareVersion."Active iBMC", I am interested.

Reply:
No worries. That oid is indexed by firmwareName which is a bit silly.

Reply:
I searched for a way to snmpwalk it (in the CLI) and did not even succeed in that ... Anyway, it works.